Apps.Lexeri/Api/LexeriUploadRequest.cs

```diff
@@ -0,0 +1,14 @@
+using Apps.Lexeri.Constants;
+using Blackbird.Applications.Sdk.Common.Authentication;
+using Blackbird.Applications.Sdk.Utils.Extensions.Sdk;
+using RestSharp;
+
+namespace Apps.Lexeri.Api;
+
+public class LexeriUploadRequest : RestRequest
```

Comment:
Not sure that you need separate class for settings multipart form data header. You can use ordinary LexeriRequest but with property "AlwaysMultipartFormData" set to true.

Reply:
The LexeriUploadRequest uses a different baseUrl than the LexeriRequest (files.lexeri.com vs. terms.lexeri.com) therefore I would like to keep using the separate LexeriUploadRequest.